integration/test/cwl_util.go

```diff
@@ -0,0 +1,87 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build linux && integration
+// +build linux,integration
+
+package test
+
+import (
+	"context"
+	"errors"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs"
+	"github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs/types"
+	"github.com/aws/aws-sdk-go/aws"
```

Comment:
Probably not a big deal but is there a reason why we can't use v2 for this lib as well?

Reply:
So, this is one I went back and forth on. The SDK uses pointers to values, and `aws` provides helper functions for that, i.e.: `aws.String(logGroupName)`. I could just change it to not use the package, by doing `&logGroupName`. I'm indifferent. There is no equivalent I found in the v2 SDK.